src/app/api/api.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { QuizComponent } from '../quiz/quiz.component';
+
+
+
```

Comment:
Is it required to leave 3 empty lines here? Wouldn't 1 be enough?

Reply:
Done